src/union/fixture.json

```diff
@@ -0,0 +1,66 @@
+{
+  "serialize": [
+    {
+      "source": [
+        ["name", 0, "0x030405060708"]
```

Comment:
According to the doc, the union element only has `index` and `item`, what does the `name` in ['name', 0, '0x030405060708'] do?

Reply:
Updated.https://github.com/Keith-CY/molecule-javascript/pull/13/commits/a9c6d6f03b495bf81f92bac7ff61733bcb260e44